doc/source/openshift.md

```diff
@@ -47,14 +48,35 @@ Use the following links to create the Watson services on IBM Cloud. Copy/paste t
 * Click on the `Resources` tab and choose `Config Maps` and then click the `Create Config Map` button.
   * Provide a `Name` for the config map.
   * Add a key named `PORT` and paste in the `8080` under `Enter a value...`.
-  * Click `Add Item` and add a key named `ASSISTANT_APIKEY` and paste in the API Key from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `ASSISTANT_URL` and paste in the URL from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `DISCOVERY_APIKEY` and paste in the API Key from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `DISCOVERY_URL` and paste in the URL from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `NATURAL_LANGUAGE_UNDERSTANDING_APIKEY` and paste in the API Key from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `NATURAL_LANGUAGE_UNDERSTANDING_URL` and paste in the URL from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `TONE_ANALYZER_APIKEY` and paste in the API Key from step 1 under `Enter a value...`..
-  * Click `Add Item` and add a key named `TONE_ANALYZER_URL` and paste in the URL from step 1 under `Enter a value...`..
+  * For each of the following key/value pairs, click `Add Item` to add the key, and then paste the value in the `Enter a value...` field. The list will depend on where you provisioned your Watson services, similar to how you would update your `.env` file.
+    * For IBM Cloud, the required keys are:
+      * `CONVERSATION_APIKEY`
+      * `CONVERSATION_URL`
+      * `WORKSPACE_ID`
+      * `DISCOVERY_APIKEY`
+      * `DISCOVERY_URL`
+      * `DISCOVERY_ENVIRONMENT_ID`
+      * `DISCOVERY_COLLECTION_ID`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_APIKEY`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_URL`
+
+    * For IBM Cloud Pak for Data, the required keys are:
+      * `CONVERSATION_AUTH_TYPE` with value `cp4d`
+      * `CONVERSATION_AUTH_URL`
+      * `CONVERSATION_APIKEY`
+      * `CONVERSATION_URL`
+      * `WORKSPACE_ID`
+      * `DISCOVERY_AUTH_TYPE` with value `cp4d`
+      * `DISCOVERY_AUTH_URL`
+      * `DISCOVERY_APIKEY`
+      * `DISCOVERY_URL`
+      * `DISCOVERY_ENVIRONMENT_ID`
+      * `DISCOVERY_COLLECTION_ID`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_AUTH_TYPE` with value `cp4d`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_AUTH_URL`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_APIKEY`
+      * `NATURAL_LANGUAGE_UNDERSTANDING_URL`
```

Comment:
yeah this part is lacking user/pass and also the disable SSL stuff

Reply:
I'd like to see it similar to this one: https://github.com/IBM/watson-speech-translator/blob/master/doc/source/openshift.md

Reply:
will use your example